src/test/app/AMM_test.cpp

```diff
@@ -2822,6 +2938,57 @@ struct AMM_test : public jtx::AMMTest
                 STAmount{USD, UINT64_C(1'010'10090898081), -11},
                 IOUAmount{1'004'487'562112089, -9}));
         }
+
+        // preflight tests
+        {
+            Env env(*this);
+            fund(env, gw, {alice, bob}, XRP(2'000), {USD(2'000)});
+            AMM amm(env, gw, XRP(1'000), USD(1'010), false, 1'000);
+            Json::Value tx = amm.bidJson(alice, 500);
+
+            {
+                auto jtx = env.jt(tx, seq(1), fee(10));
```

Comment:
This test can be moved to `testAmendment()` function and can be tested with one call:

     amm.bid(BidArg{.err = ter(temDISABLED)});